portfolio/src/main/client/src/components/controls/lightbox.ts

```diff
@@ -45,5 +46,9 @@ const lightbox: HTMLDivElement = htmlElement`
   </div>`;
 
 lightbox.addEventListener('click', hideLightbox);
+window.addEventListener('keydown', (e) => {
+  if (e.keyCode !== 27) return;
```

Comment:
KeyboardEvent.keyCode is deprecated. Use KeyboardEvent.key instead.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode